ansible/roles/rancher/tasks/cert-manager.yml

```diff
@@ -0,0 +1,50 @@
+---
+- name: Probe cert-manager Custom Resource Definition
+  command: kubectl get CustomResourceDefinition certificates.certmanager.k8s.io
+  run_once: true
+  ignore_errors: true
+  changed_when: false
+  register: crd
+  
+- name: Install cert-manager Custom Resource Definition
+  command: kubectl apply -f https://raw.githubusercontent.com/jetstack/cert-manager/release-0.9/deploy/manifests/00-crds.yaml
```

Comment:
This will not work with offline installs

Reply:
Per our sync last week, it doesn't seem like we currently have a walled-garden deploy. However, if we do we can always checkin this file locally.